### ISSUE TITLE
ITV Cache update and Use Current Webpage acting strange for me...

### ISSUE THREAD

**Issue by Mimpy** [Ascoware/get-iplayer-automator#137]
I'm on the latest version of GIA 1.11.2. MACOS 10.13.4
This happened on the previous version also.

When I launch the app, it goes through the motion of updating the ITV and BBC caches then attempts to add shows that are in my pvr queue. It MOSTLY adds the BBC shows no problem, but the ITV shows are never added. I have to manually go to file/force cache update to get the shows added. That SOMETIMES works with the ITV shows, and usually always with BBC. I even check the ITV webpage to make sure there was a new episode available. There is. When it fails, I try to use the Use Current Webpage button and it adds the show with correct title. But, when I press the Process PID button, it always changes the show to "Unknown:PID Not Found". I've just tried downloading the Unknown PID file and it is downloading. The app shows "DOWNLOADING Unknown:PID Not Found" in the lower left but no time remaining countdown clock.
The file has downloaded and it IS the correct show. 

The show I'm trying to watch is "Who's Doing the Dishes". I made the mistake of trying to remove and re add to the PVR queue and now it's also not found in the search. BUT it is on the ITV site with a new episode available from last night.

No proxy or VPN

In summary, ITV shows, FOR ME,  are not added to download queue after app startup auto cache update, nor are they found in show PVR search. Only able to download using webpage button and the meta is missing.
Thanks for the hard work and the community help!

**Comment by Mimpy** [comment]
I did a clean and thorough uninstall and re-installed the app. It appears to be working for now. Maybe there were too many version updates on top of updates and it became corrupted? I have had it installed a very long time so it's been updated many many times. Anyway, if there are issues, I'll report it. Thanks.

**Comment by Mimpy** [comment]
I see I wasn't alone in this. The re-install did not work. Who's Doing the Dishes (and others) has a new episode available but not showing up in search either PVR or main window as described yesterday.

**Comment by hugofalkman** [comment]
@Mimpy  can we please only keep one issue open for one problem without duplicates. It is the same problem in issues #137, #138, and #139. As you had closed this issue #137 I asked @bevanting to close #139 as duplicate of #138.

So can you please close this issue also as a duplicate of #138.

Basically the problem description is the ITV cache does not get updated and the bypass is forcing a cache update.

**Comment by Mimpy** [comment]
Will do. I knew it was a dup but thought someone might want to know history or data from mine. Closing.
